### ISSUE TITLE
Custom Prompt Saving Fails Due to 'Quota Exceeded' Issue

### ISSUE THREAD

**Issue by jamaical82** [micz/ThunderAI#129]
### The problem

When I create more than 6 or 7 prompts (which are fairly long), the saving function stops working and gets stuck on the 'saving custom prompts' status.

![error](https://github.com/user-attachments/assets/a2a492a4-4793-489b-b787-9dd1377a281a)

### Which Operative System are you using?

windows 10 and 11

### Which version of Thunderbird are you using?

115.14 and 128.2.0

### Which version of ThunderAI has the issue?

2.1.1.

### Which integration are you using?

ChatGPT Web Interface

### Anything in the Thunderbird console logs that might be useful?

```txt
The Thunderbird console reports the error 'Quota exceeded: ItemByte'
```

**Comment by micz** [comment]
To solve this issue, it is necessary to move from `storage.sync` to `storage.local`.

To do:
- [ ] Implement new methods to store and save custom prompts in `storage.local`.
- [ ] Display the amount of space used in `storage.local` given the 5MB limit.
- [ ] Add a migration step from `storage.sync` to `storage.local` to preserve already saved prompts.

**Comment by micz** [comment]
Fixed in version 2.2.0.
